tests/test_core.py

```diff
@@ -9,6 +9,45 @@
 
 EXE_NAME = "strings.exe"
 
+# def test_invalid_vm_arg_fail():
```

Comment:
This commented test works on its own, but it breaks the JVM init with the expected error. If this test is run, all subsequent calls to jpype.startjvm will fail.  This seems to be a jpype issue not able recover from an initialization error. If this test could be run in its own subprocess, it would succeed. Is that possible with `pytest`?

Reply:
This test breaks the JVM init, **and** must run before the JVM is started. So it won't be able to run at the end. I've found a decorator that might help us here... commit pending. https://stackoverflow.com/questions/2046603/is-it-possible-to-run-function-in-a-subprocess-without-threading-or-writing-a-se .

Reply:
Sorry for our earlier misunderstanding.  A simple multiprocessing call should do the trick. Otherwise, if it is getting too complex, not having this test isn't a big deal.

Reply:
Removed the test :)